MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include requirements.txt
-include CHANGELOG.rst
\ No newline at end of file
+include CHANGELOG.rst
+include lightning.qubit.toml
+include lightning.kokkos.toml
```

Comment:
Although we settled on having separate configuration files for each Lightning simulators, I think with the monorepo work and unifying the support in all lightning sims it now makes more sense to merge all these toml files to one single toml file, `lightning.toml`. What do you think?

Reply:
@maliasadi is there ever a case where the devices could have different options/capabilities?

Reply:
I think as long as they are different devices they should have a separate toml. 1 toml per device seems reasonable.

Reply:
> @maliasadi is there ever a case where the devices could have different options/capabilities?

Yes, for example we have a ticket to implement iSWAP in Lightning-Qubit. When that goes in, it should be added to the L-Qubit toml, but not the others.

Reply:
:100:

Reply:
Happy to keep these toml files separated as planned.